.jvmopts

```diff
@@ -1,4 +1,5 @@
 -Dfile.encoding=UTF8
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
Except for directly after booting identical virtual machines (in the end of that article).